ui/src/app/views/workspace-share/component.css

```diff
@@ -1,14 +1,84 @@
-input {
-  width: 5rem;
+h3 {
+  font-family: GothamMedium;
```

Comment:
Is this a global style, or just for this page? And, if it is a global style, can we put it in `headers.css`?

Reply:
Not a global style.

Reply:
Please rebase against master (for https://github.com/all-of-us/workbench/pull/708) and double check whether these font settings are still needed (or at least they probably need updating).

Reply:
done and CSS isn't needed anymore